Modules/_zstd/compressor.c

```diff
@@ -7,24 +7,50 @@ Python module.
 
 /*[clinic input]
 module _zstd
-class _zstd.ZstdCompressor "ZstdCompressor *" "clinic_state()->ZstdCompressor_type"
+class _zstd.ZstdCompressor "ZstdCompressor *" "zstd_compressor_type_spec"
```

Comment:
I never used a type spec here as I've only seen clinic_state()->TYPE constructions so I don't know if it's correct. However, if you want to keep this, use `&zstd_compressor_type_spec` not `zstd_compressor_type_spec` as you did for the other types.

Reply:
Oops re `&`! There are other examples of _spec being used, so I think it's ok.